### PR TITLE
API 1600 support for Enclosures

### PR DESCRIPTION
**Pull request by chebroluharika**
### Description
API 1600 support for Enclosures

### Issues Resolved
NA

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -40,21 +40,21 @@
...
+|<sub>/rest/enclosures/{id}</sub>                                                         | GET      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |
+|<sub>/rest/enclosures/{id}</sub>                                                         | PATCH    |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |
+|<sub>/rest/enclosures/{id}</sub>                                                         | DELETE   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |
+|<sub>/rest/enclosures/{id}/configuration</sub>                                           | PUT      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |
+|<sub>/rest/enclosures/{id}/environmentalConfiguration</sub>                              | GET      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |
+|<sub>/rest/enclosures/{id}/environmentalConfiguration</sub>                              | PUT      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:50]: didn't find this endpoint in api docs. Please check

File: endpoints-support.md [hunk 2]
```diff
@@ -40,21 +40,21 @@
...
+|<sub>/rest/enclosures/{id}/refreshState</sub>                                            | PUT      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |
+|<sub>/rest/enclosures/{id}/script</sub>                                                  | GET      |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |
+|<sub>/rest/enclosures/{id}/sso</sub>                                                     | GET      |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:53]: Please check API docs. Didn't find this endpoint from API 500

Comment (VenkateshRavula) [reply endpoints-support.md:53]: Add necessary validations in example file if required

File: endpoints-support.md (hunk 1 above)
Comment (chebroluharika) [reply endpoints-support.md:50]: Those are available for C7000 variant

File: endpoints-support.md (hunk 2 above)
Comment (chebroluharika) [reply endpoints-support.md:53]: Those are available for C7000 variant.

**Pull request merged**